Cargo.toml

```diff
@@ -340,13 +340,13 @@ wasmcloud-provider-keyvalue-vault = { version = "*", path = "./crates/provider-k
 wasmcloud-provider-lattice-controller = { version = "*", path = "./crates/provider-lattice-controller", default-features = false }
 wasmcloud-provider-messaging-kafka = { version = "*", path = "./crates/provider-messaging-kafka", default-features = false }
 wasmcloud-provider-messaging-nats = { version = "*", path = "./crates/provider-messaging-nats", default-features = false }
-wasmcloud-provider-sdk = { version = "^0.6.0", path = "./crates/provider-sdk", default-features = false }
+wasmcloud-provider-sdk = { version = "^0.7.0", path = "./crates/provider-sdk", default-features = false }
 wasmcloud-provider-sqldb-postgres = { version = "*", path = "./crates/provider-sqldb-postgres", default-features = false }
 wasmcloud-runtime = { version = "0", path = "./crates/runtime", default-features = false }
-wasmcloud-secrets-client = { version = "0.2", path = "./crates/secrets-client", default-features = false }
-wasmcloud-secrets-types = { version = "0.2", path = "./crates/secrets-types", default-features = false }
+wasmcloud-secrets-client = { version = "^0.3.0", path = "./crates/secrets-client", default-features = false }
+wasmcloud-secrets-types = { version = "^0.3.0", path = "./crates/secrets-types", default-features = false }
```

Comment:
Not needed, but I guess smart-release just didn't know about these so I'll take it